infra/license/license-blocklist.json

```diff
@@ -0,0 +1,24 @@
+{
+    "Allowed": [
+        "0BSD",
+        "Apache-2.0",
+        "BSD-2-Clause",
+        "BSD-3-Clause",
+        "CC-BY-3.0",
+        "CC0-1.0",
+        "ISC",
+        "MIT",
+        "Unlicense",
+        "(MIT AND CC-BY-3.0)",
+        "(MIT OR CC0-1.0)"
```

Comment:
Maybe duplicated since L10 is MIT and L8 is CC0-1.0?

Reply:
Ah, I missed this!
Saying conclusion first, I will move `(MIT AND CC-BY-3.0)` and `(MIT OR CC0-1.0)` to `Warning` :)

---

Literally, your suggestion is right :)
But I didn't work for parsing those things because of following reasons.

- Risk of parsing
  - There can be various variations for multiple licenses
    - `(MIT OR CC0-1.0)`
    - `MIT OR CC0-1.0`
    - `(MIT or CC0-1.0)`
    - `MIT or CC0-1.0`
    - ...
  - "OR" and "AND" often have specific condition which needs manual verification
    - For example, if license is "MIT OR GPL-2.0" under some condition, the package license would be GPL-2.0 unless the condition is met. But parser cannot determine this.

First reason may be resolved somehow, but second reason is too important to follow the result of parser.

Reply:
I misunderstood @llFreetimell's intention.
@llFreetimell told me that license in npm web site (or other web site about node package) shows info such as L3 to L13.
An example is https://snyk.io/advisor/npm-package/excel-builder-webpacker, where its license is mentioned as `(MIT OR GPLv3)`.

> Saying conclusion first, I will move (MIT AND CC-BY-3.0) and (MIT OR CC0-1.0) to Warning :)

I agree. If legal things are not very clear, let's not allow those. :-)